config/openshift3.11/kustomization.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
Would an `openshift-common` folder with patches for the mismatched apiversions work for this?

similar to the following folder structure:
- openshift-common
	- `mutatingwebhookconfiguration.yaml`
	- `securitycontextconstraint.yaml`
	- ...
- openshift
	- kustomization with `common` + `openshift-common`
	- default CRD
- openshift 3.11
	- kustomization with `common` + `openshift-common`
	- patch to change apiversion of `mutatingwebhookconfiguration.yaml` to `v1beta1` 
	- openshirt 3.11 CRD